doc/ref_model/chapters/glossary.md

```diff
@@ -80,14 +78,24 @@ To help guide the reader, this glossary provides an introduction to the terminol
 
 - **Container Image**:	Stored instance of a container that holds a set of software needed to run an application.
 - **Container**:	A lightweight and portable executable image that contains software and all of its dependencies.
+>_*Note:*_ OCI defines **Container** as "An environment for executing processes with configurable isolation and resource limitations. For example, namespaces, resource limits, and mounts are all part of the container environment." 
+A **Container** provides operating-system-level virtualization by abstracting the “user space”. One big difference between **Containers** and **VM**s is that unlike VMs, where each **VM** is self-contained with all the operating systems components are within the **VM** package, containers "share" the host system’s kernel with other containers.
+
+- **Container Runtime**: reads the configuration files for a **Container** from a directory structure, uses that information to create a container, launches a process inside the container, and performs other lifecycle actions."
```

Comment:
As we're using Kubernetes definitions for the rest of these terms, where possible, can we do so for Container Runtime please: https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-container-runtime

Reply:
Sure. Good idea. I moved the explanation that OCI provides to the note.
@tomkivlin let me know whether this is ok for you.

Reply:
@ulikleber great, thanks